package.json

```diff
@@ -74,6 +74,8 @@
     "@rollup/plugin-json": "^6.0.0",
     "@rollup/plugin-node-resolve": "^15.0.2",
     "@rollup/plugin-typescript": "^11.1.0",
+    "@testing-library/dom": "^9.2.0",
+    "@testing-library/user-event": "^14.4.3",
```

Comment:
there is already one in the packages/render/package.json file

Both need to go in the renderer `package.json` file as they're no usecase of this library on the other part (like main)

Reply:
https://github.com/containers/podman-desktop/blob/main/packages/renderer/package.json#L20